src/main/Kotlin/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
Jetzt ist das Manifest an neuer Stelle wieder aufgetaucht. Musst es noch mal löschen.

Reply:
Hab es jetzt überall gelöscht und den Kotlin Ordner in kotlin umbenannt, 
bei git status erkennt er allerdings keine Änderungen.